src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/MSBuildIntegrationTestBase.cs

```diff
@@ -70,7 +70,7 @@ internal static string TargetFramework
             bool suppressBuildServer = false,
             string buildServerPipeName = null,
             MSBuildProcessKind msBuildProcessKind = MSBuildProcessKind.Dotnet,
-            bool runRestoreBeforeBuildOrPublish = true)
+            bool runRestoreBeforeBuildOrPublish = false)
```

Comment:
Can we remove this flag and have tests pass this in as an argument? It really doesn't warrant being a first class feature of the test framework.